### ISSUE TITLE
Build warnings when using static library with Swift

### ISSUE THREAD

**Issue by Silkjaer** [onesky/plugin-ios-ota#6]
E.g. "Could not resolve external type c:objc(cs)NSObject"

Can you turn off "Precompile Prefix Header" when compiling the library?

**Comment by bretdabaker** [comment]
I have tried both CocoaPod and .framework on a swift project and didn't see any warning. I tested it on both Xcode 7 GM Seed and Xcode 7 release. 

Are you sure it's caused by the plugin?

**Comment by Silkjaer** [comment]
It is after updating to using libOneSkyOTAPlugin.a – but it's just warnings, and I can ignore them.

But at least I can now compile my project with bitcode enabled!

```
while processing /XXX/Vendors/OneSkyOTAPlugin/libOneSkyOTAPlugin.a(OneSkyOTAHelper.o):
warning: /Users/bretcheng/Library/Developer/Xcode/DerivedData/ModuleCache/1QDOBMFLRFGI2/Foundation-3DFYNEBRQSXST.pcm: No such file or directory
while processing /XXX/Vendors/OneSkyOTAPlugin/libOneSkyOTAPlugin.a(OneSkyOTAHelper.o):
warning: /Users/bretcheng/Library/Developer/Xcode/DerivedData/ModuleCache/1QDOBMFLRFGI2/Foundation-3DFYNEBRQSXST.pcm: No object file for requested architecture
while processing /XXX/Vendors/OneSkyOTAPlugin/libOneSkyOTAPlugin.a(OneSkyOTAHelper.o):
warning: /Users/bretcheng/Library/Developer/Xcode/DerivedData/ModuleCache/1QDOBMFLRFGI2/ObjectiveC-1RQA2RIXJIE19.pcm: No such file or directory
while processing /XXX/Vendors/OneSkyOTAPlugin/libOneSkyOTAPlugin.a(OneSkyOTAHelper.o):
warning: /Users/bretcheng/Library/Developer/Xcode/DerivedData/ModuleCache/1QDOBMFLRFGI2/ObjectiveC-1RQA2RIXJIE19.pcm: No object file for requested architecture
while processing /XXX/Vendors/OneSkyOTAPlugin/libOneSkyOTAPlugin.a(OneSkyOTAHelper.o):
warning: Could not resolve external type c:objc(cs)NSObject
```

**Comment by bretdabaker** [comment]
It seems to be a bug of Xcode where it tries to create dSYM files for debug builds, have you tried this solution? https://forums.developer.apple.com/thread/17921#58073

I also read that it only happens to projects upgrading from Xcode 6 to 7, it would be great if you could provide a minimal project that builds with the warnings, if not then let me try to reproduce the problem.

**Comment by bretdabaker** [comment]
Fixed in [0.9.4](https://github.com/onesky/plugin-ios-ota/releases/tag/0.9.4)

**Comment by farzadshbfn** [comment]
Can i ask how did you manage to fix'em?
